Recrutify/Recrutify.Services/DTOs/StaffGroupDTO.cs

```diff
@@ -1,8 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
```

Comment:
Can remove this file

Reply:
I can’t because it’s used into GetByGroupRoleAsync

Reply:
Where do you use GetByGroupRoleAsync?